.github/build.in.yml

```diff
@@ -156,6 +156,10 @@ jobs:
           run: |
             bash -o pipefail -c "env LEAN_ABORT_ON_PANIC=1 lake build --wfail -KCI"
 
+      - name: print the sizes of the oleans
+        run: |
+          du .lake/build/lib/Mathlib || echo "This code should be unreachable"
```

Comment:
Noting down for myself: this is using Unix paths --- this is fine as this only runs on the builders, which run Ubuntu. (Up to you if you want to leave a comment about this.)

Reply:
True. The existence of du itself is probably a Unix-ism also.